app/Models/IndependentActivity.php

```diff
@@ -131,4 +137,20 @@ public function organization()
     {
         return $this->belongsTo('App\Models\Organization');
     }
+
+    /**
+     * Maps the indexing integer value and returns the text
+     * @return string|null
+     */
+    public function getIndexingTextAttribute(){
```

Comment:
The start bracket of the function should be on the next line as other functions have.